hudi-common/src/test/java/org/apache/hudi/common/util/TestSerializationUtils.java

```diff
@@ -52,12 +58,33 @@ public void testSerDeser() throws IOException {
     verifyObject(new LinkedList<>(Arrays.asList(2, 3, 5)));
   }
 
+  @Test
+  public void testAvroUtf8SerDe() throws IOException {
+    byte[] firstBytes = SerializationUtils.serialize(new Utf8("test"));
+    // 4 byte string + 3 bytes length (Kryo uses variable-length encoding)
+    assertEquals(firstBytes.length, 7);
```

Comment:
/nit expected value goes first

Reply:
Ah, fixing this for existing tests, and fallen prey for this myself